frontend/src/Frontend/UI/DeploymentSettings.hs

```diff
@@ -621,12 +621,12 @@ capabilityInputRow
   -> m (Dynamic t (Maybe AccountName))
   -> m (CapabilityInputRow t)
 capabilityInputRow mCap mkSender = elClass "tr" "table__row" $ do
-  (empty, parsed) <- el "td" $ mdo
+  (empty, parsed) <- elClass "td" "table__cell_padded" $ mdo
     cap <- uiInputElement $ def
       & inputElementConfig_initialValue .~ foldMap (renderCompactText . _dappCap_cap) mCap
       & initialAttributes .~
         "placeholder" =: "(module.capability arg1 arg2)" <>
-        "class" =: "input_width_full" <>
+        "class" =: (maybe id (const (<> " input_transparent")) mCap) "input_width_full" <>
```

Comment:
```suggestion
        "class" =: (bool id (<> " input_transparent") (isJust mCap)) "input_width_full" <>
```

Reply:
I don't like `bool` because I have to open hoogle every single time I see it